src/design/optForce/analizeOptForceSol.m

```diff
@@ -0,0 +1,131 @@
+function [maxGrowthRate, minTarget, maxTarget] = analizeOptForceSol(model, targetRxn, solution, relax, tol)
```

Comment:
Will you have to "publish" after moving but before editing the file?
Or how does it detect the file move? 
because analizeOptForceSol is the original testOptForceSol file.